fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/BaseRepeatedField.java

```diff
@@ -36,6 +36,10 @@ public BaseRepeatedField(@Nonnull String fieldName, Field.OneOfThemEmptyMode emp
         this.emptyMode = emptyMode;
     }
 
+    public Field.OneOfThemEmptyMode getEmptyMode() {
```

Comment:
Missing `@Nonnull` annotation

Reply:
Added (also on the field and also added release note).